### DISCUSSION TITLE
Add folders, labels or tags

### DISCUSSION THREAD

**Discussion by beamkiller** [pschlan/cron-job.org#142] [Ideas]
Hi,

Can you please add someting like folders and labels or tags so we can search in Cron Jobs faster.

Thanks.

**Comment by pschlan** [comment]
Hi,

good suggestion - I'll see what I can do.

Thanks,

Patrick

**Reply by beamkiller** [reply]
Thank you. It would keep things more organized. If you have more CronJobs it's hard to track them correctly :)
My solution for now is that I add the domain name to the Cron Job name, so I can search for it with CTRL+F.

**Comment by pschlan** [comment]
See also: #46

**Comment by tobeyadr** [comment]
We manage lots of cron jobs for clients, this would be huge for us.

**Comment by pschlan** [comment]
Now working on this :)

**Comment by pschlan** [comment]
Now available in beta stage. Feel free to give it a try at https://console.beta.cron-job.org/ and share your feedback. To get started, find the folder manager at "Cron jobs" -> "Manage folders". You can move jobs by editing them and changing the folder in the job editor.

A bulk move feature will follow later.

**Reply by pschlan** [reply]
Bulk move now added as well.
